index.bs

```diff
@@ -646,6 +655,7 @@ When this method is invoked, the user agent MUST run the following steps:
   1. If |newState|'s {{XRRenderStateInit/inlineVerticalFieldOfView}} is set and |session| is an [=immersive session=], throw an {{InvalidStateError}} and abort these steps.
   1. Let |activeState| be |session|'s [=active render state=].
   1. If |session|'s [=pending render state=] is <code>null</code>, set it to a copy of |activeState|.
+  1. If |newState|'s {{XRRenderStateInit/layers}}'s value is not <code>null</code>, run [=update the pending layers state=] with |session| and |newState|.
```

Comment:
I just realized: we want the patch to be able to null out `layers` when `baseLayer` is set, right? in that case this step should be unconditional, and the `update the pending layers state` patch in the layers spec should handle "layers set" and "layers not set" differently

Reply:
Yes, we should always call that function so when we patch it in the layers spec, we can do the conditional checking.

Reply:
@Manishearth I just updated the patch.
Hard to keep my head around a patch of a patch of a patch :-)